### PR TITLE
Setup README.md

### PR DESCRIPTION
**Pull request by marshallpete**
## Description

Add the following to the README.md

- Table of Contents
- Overview
  - Key Features:
- Installation
  - npm
  - yarn
- Usage
  - Example
- Spectrum (Adobe Design System) Integration
- API
- Contributing
- License
  - Apache License 2.0 Summary
- Roadmap

## Motivation and Context

Need a detailed README for this project

## How Has This Been Tested?

Doc only. No tests

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,6 +12,153 @@
...
+
+-   **Built on Vega:** `react-spectrum-charts` is built on top of Vega, a robust and versatile visualization grammar for expressive, concise, and interactive data visualizations. This foundation ensures both flexibility and performance.
+
+`react-spectrum-charts` simplifies the charting process, making it accessible to a broader audience, including developers who may not have prior experience in data visualization. With just a few lines of code, you can create charts that meet the Adobe design system's standards, enriching your React applications with informative and visually appealing data representations.
+
+## Installation
+
+-   Instructions for installing your library.
+-   Include code snippets for installation via npm or yarn.
+-   Mention any peer dependencies.
```
Comment (c-lamoureux) [inline README.md:73]: I'm assuming these were left in on accident.

File: README.md [hunk 2]
```diff
@@ -12,6 +12,153 @@
...
+The Adobe design system has detailed guidelines for charting fundamentals, color selection for charts as well as design guidelines for the primary chart components.
+
+`react-spectrum-charts` is the react implementation of these guidelines.
+
+-   [Fundamentals](https://spectrum.adobe.com/page/data-visualization-fundamentals/)
+-   [Color](https://spectrum.adobe.com/page/color-for-data-visualization/)
+-   Components
+    -   [Area](https://spectrum.corp.adobe.com/page/area-chart/)
+    -   [Bar](https://spectrum.corp.adobe.com/page/bar-chart/)
+    -   [Line](https://spectrum.corp.adobe.com/page/line-chart/)
```
Comment (c-lamoureux) [inline README.md:124]:
```suggestion
    -   [Line](https://spectrum.adobe.com/page/line-chart/)
```

Comment (c-lamoureux) [inline README.md:124]:
```suggestion
    -   [Area](https://spectrum.adobe.com/page/area-chart/)
    -   [Bar](https://spectrum.adobe.com/page/bar-chart/)
    -   [Line](https://spectrum.adobe.com/page/line-chart/)
```

File: README.md [hunk 3]
```diff
@@ -12,6 +12,153 @@
...
+
+## API
+
+[API Documentation](https://github.com/adobe/react-spectrum-charts/wiki)
+
+## Storybook
+
+This library has an extensive [Storybook](opensource.adobe.com/react-spectrum-charts/) with stories for every component as most props. You can use the controls tab in storybook to alter component props and see how that alters the chart real time.
+
+It is also possible to view the source code for any story by selecting the "Docs" tab, scrolling to the desired story and then selecting "Show code". This is helpful for getting seeing the full API for any story in storybook.
```
Comment (c-lamoureux) [inline README.md:134]:
```suggestion
It is also possible to view the source code for any story by selecting the "Docs" tab, scrolling to the desired story and then selecting "Show code". This is helpful for seeing the full API for any story in storybook.
```

File: README.md [hunk 4]
```diff
@@ -12,6 +12,153 @@
...
+
+-   **More Information**: For a full and detailed explanation of the Apache License 2.0, please refer to the [official license document](https://www.apache.org/licenses/LICENSE-2.0).
+
+By using **react-spectrum-charts**, you agree to comply with the terms and conditions of the Apache License 2.0.
+
+Please review the [LICENSE](LICENSE) file for the complete text of the license.
+
+## Roadmap
+
+The roadmap for this project is tracked in github projects. You must be a member of the Adboe org to see the roadmap.
```
Comment (c-lamoureux) [inline README.md:162]:
```suggestion
The roadmap for this project is tracked in github projects. You must be a member of the Adobe org to see the roadmap.
```

File: README.md [hunk 5]
```diff
@@ -12,6 +12,153 @@
...
+
+#### yarn
+
+```
+yarn add @adobe/react-spectrum-charts
+```
+
+## Usage
+
+The `react-spectrum-charts` is designed in a way that makes composing charts similar to composing any other app content in JSX.
```
Comment (c-lamoureux) [inline README.md:89]:
```suggestion
`react-spectrum-charts` is designed in a way that makes composing charts similar to composing any other app content in JSX.
```

File: README.md [hunk 6]
```diff
@@ -12,6 +12,153 @@
...
+    -   [Bar](https://spectrum.corp.adobe.com/page/bar-chart/)
+    -   [Line](https://spectrum.corp.adobe.com/page/line-chart/)
+
+## API
+
+[API Documentation](https://github.com/adobe/react-spectrum-charts/wiki)
+
+## Storybook
+
+This library has an extensive [Storybook](opensource.adobe.com/react-spectrum-charts/) with stories for every component as most props. You can use the controls tab in storybook to alter component props and see how that alters the chart real time.
```
Comment (c-lamoureux) [inline README.md:132]:
```suggestion
This library has an extensive [Storybook](opensource.adobe.com/react-spectrum-charts/) with stories for every component and most props. You can use the controls tab in storybook to alter component props and see how that alters the chart real time.
```

Comment (c-lamoureux) [reply README.md:132]: or `as well as`

File: README.md [hunk 7]
```diff
@@ -12,6 +12,153 @@
...
+    return (
+        <Prism data={myChartData}>
+            <Axis position="bottom" />
+            <Axis position="left" />
+            <Bar type="stacked" color="series" />
+            <Legend />
+        </Prism>
+    )
+}
+```
```
Comment (c-lamoureux) [inline README.md:111]: Do you think a screenshot here would be helpful? Or would it be too much? I'm undecided so I'm curious what your opinion is.

**Comment by c-lamoureux** [comment]
We should probably have a support section, but we don't have an established template for submitting issues yet. We could include a placeholder for now or hold off until we've ironed things out some more.

File: README.md (hunk 1 above)
Comment (marshallpete) [reply README.md:73]: Yeah...

File: README.md (hunk 2 above)
Comment (marshallpete) [reply README.md:124]: Good catch

File: README.md (hunk 7 above)
Comment (marshallpete) [reply README.md:108]: I might add one later. I don't think I will right now.

**Comment by marshallpete** [comment]
We actually do have issue templates so I added a support section

**Comment by c-lamoureux** [comment]
Awesome job writing this up! It looks great!

**Review by c-lamoureux — APPROVED** [review]

**Pull request merged**